CMPUT301W23T41/SnailsCurlUp
# Map Fragment and Functionality

Added in everything related to the map feature. This includes:
- A map widget powered by the Google Maps API
- A connection to the Google Maps API (plus all of the API key and validation nonsense)
- A working connection to the database that reads in QR codes and plots them on the map

To test for this pull request:
1. Send me the SHA1 hash of your Android Studio installation so I can add it to the credentials of the API key (map won't work without this).
2. I will send you a line of code containing the secret API key that needs to be added to the `local.properties` file you should have in your repository (it should not be tracked by Git though, beware!)
